thoth/user_api/api_v1.py

```diff
@@ -409,6 +409,7 @@ def post_advise_python(
     parameters["stack_info"] = parameters["input"].pop("stack_info", None)
     parameters["kebechet_metadata"] = parameters["input"].pop("kebechet_metadata", None)
     parameters["labels"] = parameters["input"].pop("labels", None)
+    parameters["package_update_info"] = parameters["input"].pop("package_update_info", None)
```

Comment:
Should the parameter be part of `_ADVISE_PROTECTED_FIELDS` to require a token for this input?

Reply:
I don't think it needs to be. Users should be free to use this feature if they would like.

Reply:
Not sure about a use case when an external user would trigger an update of a package manually on user-api. I think this update should come from the system.